internal/iam/command_pool_create.go

```diff
@@ -57,13 +58,18 @@ func (c *poolCommand) create(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	assignedResourceOwner, err := cmd.Flags().GetString("resource-owner")
```

Comment:
We like to keep a 1-1 mapping of flag name to variable name to avoid confusion
```suggestion
	resourceOwner, err := cmd.Flags().GetString("resource-owner")
```

Reply:
Looks like you may not have pushed this change.